### ISSUE TITLE
Script breaks at importing librosa

### ISSUE THREAD

**Issue by moehr1z** [tsurumeso/vocal-remover#134]
When running `python3 inference.py --input path/to/an/audio/file` the script throws the following error:

```
Traceback (most recent call last):
  File "/root/vocal-remover-v5.0.3(1)/vocal-remover/inference.py", line 4, in <module>
    import librosa
  File "/usr/local/lib/python3.11/dist-packages/librosa/__init__.py", line 211, in <module>
    from . import core
  File "/usr/local/lib/python3.11/dist-packages/librosa/core/__init__.py", line 9, in <module>
    from .constantq import *  # pylint: disable=wildcard-import
    ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/librosa/core/constantq.py", line 1059, in <module>
    dtype=np.complex,
          ^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/numpy/__init__.py", line 305, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'complex'.
`np.complex` was a deprecated alias for the builtin `complex`. To avoid this error in existing code, use `complex` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.complex128` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations. Did you mean: 'complex_'?
```

Edit:
Installing librosa not via the requirements file, but installing directly the latest version fixes it.
So the requirements.txt probably has to be updated.

**Comment by CrazyChildren** [comment]
same issue, using `pip install upgrade librosa` will resolve this problem

**Comment by tsurumeso** [comment]
LGTM

**Comment by moehr1z** [comment]
Fixed with #136
